src/EditorConfigCompletionProvider.ts

```diff
@@ -213,7 +229,28 @@ class EditorConfigCompletionProvider implements CompletionItemProvider {
 		values: string[]
 	): CompletionItem[] {
 		return values.map(
-			value => new CompletionItem(value, CompletionItemKind.Value)
+			value => {
+				// basic info
+				const completionItem = new CompletionItem(value, CompletionItemKind.Value);
+
+				// sort predefined values to specific order
+				switch (value) {
+					case 'true':
+						completionItem.sortText = '1'; // always first
+						break;
+					case 'false':
+						completionItem.sortText = '2'; // always second
+						break;
+					case 'unset':
+						completionItem.sortText = '9'; // always the last
+						break;
+					default:
+						completionItem.sortText = '3' + value; // after true or false, keeping the natural order
```

Comment:
`+ value` has the purpose to keep the natural order.

For example, in `end_of_line`, values will be `3cr`, `3crlf` and `3lf`. If I don't append the value, all of them will have `3` as `sortText` and the order will probably break.